fbcnms-packages/fbcnms-ui/master/Organizations.js

```diff
@@ -9,47 +9,145 @@
  */
 
 import type {OrganizationPlainAttributes} from '@fbcnms/sequelize-models/models/organization';
+import type {UserType} from '@fbcnms/sequelize-models/models/user.js';
 import type {WithAlert} from '@fbcnms/ui/components/Alert/withAlert';
 
-import Button from '@fbcnms/ui/components/design-system/Button';
-import DeleteIcon from '@material-ui/icons/Delete';
+import ActionTable from '../components/ActionTable';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
 import EditUserDialog from '@fbcnms/ui/components/auth/EditUserDialog';
-import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
 import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
 import NestedRouteLink from '@fbcnms/ui/components/NestedRouteLink';
 import OrganizationDialog from './OrganizationDialog';
-import Paper from '@material-ui/core/Paper';
 import PersonAdd from '@material-ui/icons/PersonAdd';
+import PersonIcon from '@material-ui/icons/Person';
 import React from 'react';
-import Table from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import TableCell from '@material-ui/core/TableCell';
-import TableHead from '@material-ui/core/TableHead';
-import TableRow from '@material-ui/core/TableRow';
+import Text from '../components/design-system/Text';
 import axios from 'axios';
-import renderList from '@fbcnms/util/renderList';
 import withAlert from '@fbcnms/ui/components/Alert/withAlert';
-import {Link, Route} from 'react-router-dom';
+
+import {Route} from 'react-router-dom';
 import {makeStyles} from '@material-ui/styles';
 import {useAxios, useRouter} from '@fbcnms/ui/hooks';
-import {useCallback, useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
 import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
 import {useRelativePath, useRelativeUrl} from '@fbcnms/ui/hooks/useRouter';
 
 export type Organization = OrganizationPlainAttributes;
 
+const ORGANIZATION_DESCRIPTION =
+  '            Multiple organizations can be independently managed, each with access to their own networks. As a host user, you can create and manage organizations here. You can also create users for these organizations.';
```

Comment:
nit: For legibility, this constant should be assigned this value over multiple lines
```suggestion
const ORGANIZATION_DESCRIPTION =
  'Multiple organizations can be independently managed, each with access to their own networks. ' +
  'As a host user, you can create and manage organizations here. You can also create users for these organizations.';
```